modules/engine/src/listeners.ts

```diff
@@ -542,6 +546,55 @@ export async function resolveExistingWithdrawals(
   );
 }
 
+export async function submitUnsubmittedWithdrawals(
```

Comment:
this method needs to be aware of who *intended* to submit the withdrawal originally. or are you assuming that if it is a bob there will be no stored transactions to pull + submit?